charms/katib-db-manager/src/charm.py

```diff
@@ -277,13 +277,13 @@ def _get_relational_db_data(self) -> dict:
             if not val:
                 continue
             try:
-                db_data["db_name"] = self._database_name
+                db_data["db_name"] = "mysql"
```

Comment:
```suggestion
                db_data["db_name"] = "mysql"  # this is DB type (mysql/postgres)
```

Reply:
I will change the description in our dict to be clear.